Packs/ThreatConnect/Integrations/ThreatConnect_v3/ThreatConnect_v3.yml

```diff
@@ -0,0 +1,2181 @@
+      required: true
+    description: Get Owner for Indicator
+    name: tc-get-indicator-owners
+  - arguments:
+    - description: The ID of the group.
+      name: group_id
+      required: true
+    description: The group report to download in PDF format.
+    name: tc-download-report
+    outputs:
+    - contextPath: File.Size
+      description: The size of the file.
+      type: Number
+    - contextPath: File.SHA1
+      description: The SHA1 hash of the file.
+      type: String
+    - contextPath: File.SHA256
+      description: The SHA256 hash of the file.
+      type: String
+    - contextPath: File.Name
+      description: The name of the file.
+      type: String
+    - contextPath: File.SSDeep
+      description: The ssdeep hash of the file.
+      type: String
+    - contextPath: File.EntryID
+      description: The entry ID of the file.
+      type: String
+    - contextPath: File.Info
+      description: The information of the file.
+      type: String
+    - contextPath: File.Type
+      description: The type of the file.
+      type: String
+    - contextPath: File.MD5
+      description: The MD5 hash of the file.
+      type: String
+    - contextPath: File.Extension
+      description: The extension of the file.
+      type: String
+  - arguments:
+    - default: true
+      description: The ID of the group.
+      name: id
+      required: true
+    - description: Custom fields for the group.
+      name: custom_field
+    - description: A comma-separated list of The tags applied to the threat.
+      name: tags
+    - auto: PREDEFINED
+      description: The security label applied to the threat. 
+      name: security_label
+      predefined:
+      - TLP:RED
+      - TLP:GREEN
+      - TLP:AMBER
+      - TLP:WHITE
+    - description: An ID to associate a group by.
+      name: associated_group_id
+    - description: An ID to associate an indicator by.
+      name: associated_indicator_id
+    - auto: PREDEFINED
+      description: The type of update to the group metadata(associated indicators, attributes,tags etc...).
+      name: security_label
+      predefined:
+      - append
+      - delete
+      - replace
+    - description: The value of the attribute to associate.
+      name: attribute_value
+    - description: The type of the attribute to associate.
+      name: attribute_type
+    description: Updates a group.
+    name: tc-update-group
+    outputs:
+    - contextPath: TC.Group.Name
+      description: The name of the group.
+      type: string
+    - contextPath: TC.Group.Owner
+      description: The owner of the group.
+      type: string
+    - contextPath: TC.Group.Tag
+      description: The tag of the group.
+      type: string
+    - contextPath: TC.Group.SecurityLevel
+      description: The security label of the group.
+      type: string
+    - contextPath: TC.Group.ID
+      description: The ID of the group.
+      type: string
+  dockerimage: demisto/threatconnect-py3-sdk:1.0.0.23423
```

Comment:
Do we still need a special docker or can we use the `demisto/python3` docker image?